examples/vocab/script_vocab.py

```diff
@@ -0,0 +1,113 @@
+from collections import OrderedDict
+import torch
+from torchtext.experimental.vocab import Vocab
+from typing import List, Optional
+
+
+class ScriptVocab(Vocab):
```

Comment:
How about the special symbols?
https://github.com/facebookresearch/pytext/blob/75acf0d0508e1b75346ba0950b2432b8bb53af40/pytext/torchscript/vocab.py#L24-L28

Reply:
I added these now!